### PR TITLE
Add the privacy policy to the menu

### LINKED ISSUE #650: Add privacy policy to the main menu
## Description

We should add the WP.org privacy policy to the Openverse menu to:

- make it easier for users to find
- make it clear that the wp.org privacy policy applies to Openverse

Additionally, future iframe changes might potentially interfere with the access to the wp.org footer, which is the main way to access this privacy policy.

## Resolution

- [x] 🙋 I would be interested in resolving this bug.

### PR DESCRIPTION
**Pull request by zackkrida**
## Fixes

Fixes #650 by @zackkrida 

## Description

Adds a link to WP.org's privacy policy to the menu.

## Checklist

- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [ ] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [ ] My commit messages follow [best practices][best_practices].
- [ ] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by sarayourfriend** [comment]
@zackkrida did you mean to remove `openverse.pot`?

**Comment by zackkrida** [comment]
Fixed, thanks @sarayourfriend! I conflated it with the various `.json` files we don't want to track.

**Review by sarayourfriend — APPROVED** [review]
Seems like this should be an acceptable stop-gap until we can get the footer working correctly.

**Comment by panchovm** [comment]
Can we set a min-width for the menu popover? It looks too narrow.

CleanShot 2022-01-20 at 15 19 43@2x
 
A `200px` width like the mockup in [Views page](https://www.figma.com/file/GIIQ4sDbaToCfFQyKMvzr8/Openverse-Design-Library?node-id=751%3A2800) looks good to me.

CleanShot 2022-01-20 at 15 23 15@2x

**Comment by zackkrida** [comment]
@panchovm sure, I made a PR for that here: https://github.com/WordPress/openverse-frontend/pull/658

File: src/components/VHeader/VSearchBar/VSearchBar.vue [hunk 1]
```diff
@@ -12,6 +12,7 @@
       type="search"
       name="q"
       :model-value="searchText"
+      v-on="{ blur: $listeners.blur, focus: $listeners.focus }"
```
Comment (krysal) [inline src/components/VHeader/VSearchBar/VSearchBar.vue:15]: It seems that there are several unrelated changes and they are causing the app to not work properly. Probably only because of this?

Comment (zackkrida) [reply src/components/VHeader/VSearchBar/VSearchBar.vue]: Ah sorry @krysal by branch was based on another PR—fixed!

**Review by krysal — APPROVED** [review]
Nice! No problem, thanks.
